docker-compose.yml

```diff
@@ -33,6 +33,8 @@ services:
       - "9992:9992"
     environment:
       - NODE_ENV=development
+    volumes:
+      - ./:/srv:rw,z
```

Comment:
```suggestion
```

This should be redundant with the app config later.

Reply:
this has been removed